tests/tensorflow_autolog/test_tensorflow2_autolog.py

```diff
@@ -553,14 +553,28 @@ def input_fn(features, labels, training=True, batch_size=256):
         feature_spec[feature] = tf.Variable([], dtype=tf.float64, name=feature)
 
     receiver_fn = tf.estimator.export.build_raw_serving_input_receiver_fn(feature_spec)
-    classifier = tf.estimator.DNNClassifier(
-        feature_columns=my_feature_columns,
-        # Two hidden layers of 10 nodes each.
-        hidden_units=[30, 10],
-        # The model must choose between 3 classes.
-        n_classes=3,
-        model_dir=directory,
-    )
+
+    # If flag set to true, then use the v1 classifier that extends Estimator
+    # If flag set to false, then use the v2 classifier that extends EstimatorV2
+    if use_v1_estimator:
+        classifier = tf.compat.v1.estimator.DNNClassifier(
```

Comment:
Based on classifier that extends Estimator:

see: [DNNClassifier](https://github.com/tensorflow/estimator/blob/master/tensorflow_estimator/python/estimator/canned/dnn.py#L770-L771)